R/write.R

```diff
@@ -73,6 +76,7 @@ xportr_write <- function(.df,
 
   assert_data_frame(.df)
   assert_string(path)
+  checkmate::assert_numeric(max_size_gb, null.ok = TRUE)
```

Comment:
```suggestion
assert_numeric(max_size_gb, null.ok = TRUE)
```

Reply:
Thanks, updated

Reply:
It needs to get placed in this file with its other friends - https://github.com/atorus-research/xportr/blob/main/R/xportr-package.R

Reply:
Thanks, updated.